app/controllers/v0/sessions_controller.rb

```diff
@@ -63,9 +63,9 @@ def saml_callback
         redirect_to saml_login_redirect_url(auth: 'fail', code: @sso_service.auth_error_code)
         stats(:failure)
       end
-    rescue NoMethodError
-      log_message_to_sentry('NoMethodError', :error, base64_params_saml_response: params[:SAMLResponse])
-      redirect_to saml_login_redirect_url(auth: 'fail', code: 7) unless performed?
+    rescue NoMethodError => e
```

Comment:
do we maybe want to consider logging any exception?

Reply:
Sentry will log any exception that's not rescued.